components/ActionSheet/src/MDCActionSheetController.m

```diff
@@ -261,6 +262,13 @@ - (UITableViewCell *)tableView:(UITableView *)tableView
   cell.mdc_adjustsFontForContentSizeCategory = self.mdc_adjustsFontForContentSizeCategory;
   cell.backgroundColor = self.backgroundColor;
   cell.actionFont = self.actionFont;
+  cell.actionColor = self.actionColor;
+  for (UIView *view in cell.contentView.subviews) {
+    if ([view isKindOfClass:[UIImageView class]]) {
```

Comment:
This isn't optimal but in order to set the tint color you have to wait until the view has loaded. If you do this in `layoutSubviews` of `MDCActionSheetItemTableViewCell` then it doesn't get set by default. The only other solution I could think of was a flag `imageColorHasBeenSet` in `MDCActionSheetItemTableViewCell` if that was `NO` then set the color to black.

Reply:
Can we set the tintColor on the cell's contentView? `tintColor` is a recursive API so any subviews should adopt it automatically.

Reply:
That worked.